templates/layout.html

```diff
@@ -79,126 +79,34 @@
         </ul>
       </div>
 
-      <!-- Collect the nav links, forms, and other content for toggling -->
+      {{!-- Collect the nav links, forms, and other content for toggling --}}
       <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
         <ul class="nav navbar-nav navbar-right">
           {{#if currentUser}}
-            <li class="{{#if isActiveRoute 'newCompetition'}}active{{/if}} leaf">
-              <a href="{{pathFor 'newCompetition'}}"
-                 data-toggle="tooltip" data-placement="bottom" data-container="body"
-                 title="New competition">
-                <span class="glyphicon glyphicon-plus"></span>
-              </a>
-            </li>
+            {{#with newCompetitionTab}}
+              {{> oneTab}}
```

Comment:
Oh wow. That's subtle. No idea if this would work, but could we pass the data context for `pathFor` explicitly somehow? Maybe the following?

```
{{> oneTab newCompetitionTab data=this}}
```